src/api/index.ts

```diff
@@ -22,6 +23,27 @@ export const createApp = (config: Config): express.Express => {
     throw new Error();
   });
 
+  app.use(express.static('public'));
+  app.use(express.urlencoded({ extended: true }));
+
+  app.post('/', async (req: Request, res, next) => {
+    try {
+      const input: PrintInput = req.body;
+      if (req.accepts('application/pdf')) {
+        res.setHeader('content-type', 'application/pdf');
+        if (input.attachment) {
+          res.setHeader('content-disposition', `attachment; filename="${req.body.attachment}"`);
+        }
+        res.status(200);
```

Comment:
I think that you could skip that `res.status(200)`  as it is the default (nit-picking here)